resources/client/cl_call.ts

```diff
@@ -89,7 +93,6 @@ on(`__cfx_nui:${events.PHONE_END_CALL}`, (data: any, cb: Function) => {
 
 onNet(events.PHONE_CALL_WAS_ENDED, () => {
```

Comment:
I may have missed the conversation revolving around this but whats the reason for not just attachingthe anim here, instead of adding another net event.

Reply:
It would trigger for the person who's being called regardless if they answered or not if done this way. The workaround was to create the net event and trigger it server side if the call was active.